src/couch_httpd.erl

```diff
@@ -837,7 +837,7 @@ error_info({bad_ctype, Reason}) ->
     {415, <<"bad_content_type">>, Reason};
 error_info(requested_range_not_satisfiable) ->
     {416, <<"requested_range_not_satisfiable">>, <<"Requested range not satisfiable">>};
-error_info({error, illegal_database_name, Name}) ->
+error_info({error, {illegal_database_name, Name}}) ->
     Message = "Name: '" ++ Name ++ "'. Only lowercase characters (a-z), "
```

Comment:
In case of db name validation customization this error message shouldn't be defined here since the requirements for the name could be different.